src/bp/nlu-core/slots/slot-tagger.ts

```diff
@@ -140,25 +147,29 @@ export function makeExtractedSlots(
 export default class SlotTagger {
   private _crfModelFn = ''
   private _crfTagger!: sdk.MLToolkit.CRF.Tagger
+  private _intentSlotFeatures: IntentSlotFeatures[] = []
 
   constructor(private mlToolkit: typeof sdk.MLToolkit) {}
 
-  load(crf: Buffer) {
+  load(crf: Buffer, intents: Intent<Utterance>[], entities: ListEntityModel[]) {
```

Comment:
I know we discussed about this but I'm not a fan of having multiple parameters of a pipeline component, I'd rather go with a simple, single arg for all components. But I think you realized that already